src/content/ueber-uns/abteilungen.svx

```diff
@@ -1,57 +1,49 @@
+## Kampfunterstützung
 
-### Kampfpanzer
-
-Die Soldaten der Kampfpanzer bedienen ausschließlich Fahrzeuge mit guter Panzerung und überwältigender Feuerkraft. Kampfpanzer sind besonders gut gegen feindliche Panzerverbände und befestigte Stellungen geeignet, erweisen sich jedoch als äußerst ineffektiv im urbanen Gelände. Oft agieren Kampfpanzer gemeinsam mit der mechanisierten Infanterie, um die Schwachstellen der Kampfpanzer zu kompensieren.
+### Pioniertruppe
 
-### Panzerartillerie
+Die Truppengattung der Pioniertruppe wird von ? organisiert und verwaltet. Sie fördert die Beweglichkeit und Durchhaltefähigkeit der Kampftruppen und hemmt die der feindlichen Truppen. Durch das Legen temporärer Brücken, das Räumen von Minenfeldern und das Beseitigen von Trümmern und Barrikaden stellen sie sicher, dass eigene Truppen Geländeabschnitte sicher überqueren können. Darüber hinaus führt die Pioniertruppe die Feldinstandsetzung von Fahrzeugen durch. Sie umfasst die Rollen des [Kampfpioniers](../logistik/kampfpionier/) und des [EOD](../logistik/eod/).
 
-Die Artillerie setzt Panzerhaubitzen mit schweren Geschützen ein. Sie wirken auf große Distanz mittels gelenktem oder ungelenktem Steilfeuer auf feindliche Kräfte oder Stellungen. Um eine präzise Wirkung zu erzielen erfordert es eine gute Kommunikation zu eigenen Kräften im Operationsraum, die der Artillerie den Feuerbefehl erteilen.
+### Fliegertruppe
 
-## Logistik
+Die Truppengattung der Fliegertruppe wird von ? organisiert und verwaltet. Sie versorgt die Kampftruppen mit Verbrauchsmaterial, führt die (Nach-) Verlegung von Kräften durch und liefert Luftnahunterstützung. Dazu bedienen sie [Transport- (DEAD)](../fuhrpark/transporthubschrauber) und [Kampfhubschrauber (DEAD)](../fuhrpark/kampfhubschrauber). Zur Fliegertruppe zählen die Rollen des [Transportpiloten (DEAD)](../logistik/transportpilot/) und des [Kampfpiloten (DEAD)](../logistik/kampfpilot/).
 
-Die Abteilung der Logistik wird von ? organisiert und verwaltet. Sie umfasst insgesamt drei Truppengattungen: Die Instandsetzung, die Versorgung und die Luftlogistik.
+## Einsatz- und Führungsunterstützung
 
-### Pioniertruppe
+### Sanitätsdienst
 
-Die Pioniertruppe setzt sich aus Pionieren und Kampfpionieren zusammen. Die Hauptaufgabe der Pioniere ist die Wartung, Bergung und Instandsetzung eigener Fahrzeuge. Zusätzlich kann es auch zu Spezialanwendungen kommen, wie zum Beispiel der Einsatz eines Brückenlegepanzers durch die Pioniertruppe.
+Die Truppengattung des Sanitätsdienstes wird von Emmi organisiert und verwaltet. Sie befasst sich mit der medizinischen Behandlung, Evakuierung und Endversorgung verwundeter Soldaten. Überwiegend sind Mitglieder des Sanitätsdienstes in die Infanterie eingegliedert, allerdings werden sie auch bei speziellen Unterstützungselementen wie dem beweglichen Arzttrupp (kurz BAT) oder dem MEDEVAC (MEDical EVACuation) eingesetzt. Der [Sanitäter](../sanitaetsdienst/sanitaeter/) ist die vielfältige Rolle des Sanitätsdienstes.
```

Comment:
```suggestion
Die Truppengattung des Sanitätsdienstes wird von Emmi organisiert und verwaltet. Sie befasst sich mit der medizinischen Behandlung, Evakuierung und Endversorgung verwundeter Soldaten. Überwiegend sind Mitglieder des Sanitätsdienstes in die Infanterie eingegliedert, allerdings werden sie auch bei speziellen Unterstützungselementen wie dem beweglichen Arzttrupp (kurz BAT) oder dem MEDEVAC (<u>med</u>ical <u>evac</u>uation) eingesetzt. Der [Sanitäter](../sanitaetsdienst/sanitaeter/) ist die vielfältige Rolle des Sanitätsdienstes.
```

Reply:
I simply used the acronyms definition as done by [Wikipedia](https://de.wikipedia.org/wiki/MedEvac).

Committed your suggested changes as of 706c576e1e342c2d6765a8d7dae7e9a045577a49.